docs/_includes/projects/docs/templates/compose/my-page.js

```diff
@@ -1,21 +1,36 @@
-import { LitElement, html } from 'lit-element';
+import {LitElement, html} from 'lit-element';
 
 class MyPage extends LitElement {
+  static get properties() {
+    return {
+      article: {
+        attribute: false,
+      },
+    };
+  }
+
+  constructor() {
+    super();
+    this.article = {
+      title: 'My Nifty Article',
+      text: 'Some witty text.',
+    };
+  }
+
   render() {
     return html`
-      ${this.headerTemplate}
-      ${this.articleTemplate}
-      ${this.footerTemplate}
+      ${this.headerTemplate(this.article.title)}
```

Comment:
Same comment with the argument as above.

Reply:
Fixed, thanks!